### PR TITLE
#311 code coverage PlayerHighlightComponent and fix path

### PR DESCRIPTION
**Pull request by 86LAK**
Wrote more junit testing for the PLayerHighlightCOmponent to boost code coverage. and fixed hard coded string.

### CODE REVIEW

File: source/core/src/main/com/csse3200/game/components/player/PlayerHighlightComponent.java [hunk 1]
```diff
@@ -18,6 +18,8 @@ public class PlayerHighlightComponent extends Component {
     // isMuted is a boolean that is true when the player is muted
     private boolean isMuted;
 
+    private static final String  TEXTURE_PATH = "images/yellowSquare.png";
+
```
Comment (JTrenerry) [inline source/core/src/main/com/csse3200/game/components/player/PlayerHighlightComponent.java:22]: Is this really needed?

File: source/core/src/test/com/csse3200/game/components/player/PlayerHighlightComponentTest.java [hunk 2]
```diff
@@ -130,5 +130,14 @@ public void updatePosition() {
...
 
+        playerHighlight.update();
+        assertEquals(playerHighlight.getPosition(), new Vector2(0.5f,0.5f));
+    }
+
+    @Test
+    public void getTexturePath() {
+        assertEquals("images/yellowSquare.png", playerHighlight.getComponent(PlayerHighlightComponent.class).getTexturePath());
     }
+
```
Comment (JTrenerry) [inline source/core/src/test/com/csse3200/game/components/player/PlayerHighlightComponentTest.java:141]: Why is this here?

Comment (86LAK) [reply source/core/src/test/com/csse3200/game/components/player/PlayerHighlightComponentTest.java:141]: :)

File: source/core/src/main/com/csse3200/game/components/player/PlayerHighlightComponent.java (hunk 1 above)
Comment (86LAK) [reply source/core/src/main/com/csse3200/game/components/player/PlayerHighlightComponent.java:22]: Quality of life

**Review by JTrenerry — APPROVED** [review]
